src/chrome/content/rules/Test-IPv6.cz.xml

```diff
@@ -1,52 +1,21 @@
 <!--
-	For rules causing false/broken MCB, see Test-IPv6.cz-falsemixed.xml.
-
-
-	Partially covered subdomains:
-
-		- (www.) *
-
-	* Avoiding false/broken MCB, rest handled in Test-IPv6.cz-falsemixed.xml
-
-
-	Fully covered subdomains:
-
-		- ds
-		- ipv4
-		- ipv6
-
-
-	Mixed content:
-
-		- Scripts, on ^ from:
-
-			- ds *
-			- ipv4 *
-			- ipv6 *
-
-		- Images on ^ from ds *
-
-	* Secured by us
-
-
-	NB: We secure all resources, and thus
-	-falsemixed should be merged for Ffx 24.
+	Invalid certificate:
+		badsig.dnssec.test-ipv6.cz
 
 -->
-<ruleset name="Test-IPv6.cz (partial)">
+<ruleset name="Test-IPv6.cz">
 
 	<target host="test-ipv6.cz" />
-	<target host="*.test-ipv6.cz" />
-		<!--
-			Avoid false/broken MCB:
-						-->
-		<exclusion pattern="^http://(?:www\.)?test-ipv6\.cz/(?!\w+\.css|\w+\.js|images/|ipv6widget/|site/)" />
-
-
-	<securecookie host="^(?!www\.).+\.test-ipv6\.cz$" name=".+" />
+	<target host="www.test-ipv6.cz" />
+	<target host="a.test-ipv6.cz" />
+	<target host="ds.test-ipv6.cz" />
+	<target host="ipv4.test-ipv6.cz" />
+	<target host="v4.test-ipv6.cz" />
+	<target host="www4.test-ipv6.cz" />
```

Comment:
You missed all IPv6 targets:
```
aaaa.test-ipv6.cz
ds.v6ns.test-ipv6.cz
ipv4.test-ipv6.cz
ipv6.test-ipv6.cz
mtu1280.test-ipv6.cz
v6.test-ipv6.cz
www6.test-ipv6.cz

Reply:
They were failing over both `http` and `https` for me but perhaps that's due to my setup (?)

Reply:
You probably don't have IPv6, they WFM.

Reply:
Still missing: `ds.v6ns.test-ipv6.cz`

Reply:
This subdomain doesn't exist for me (even on a computer with IPv6). I can add it if you want but I need you to tell me where. As a target?

Reply:
It works for me on IPv4 and IPv6. Can you try to add it as target?